librz/core/cbin.c

```diff
@@ -456,7 +456,7 @@ RZ_API bool rz_core_bin_apply_config(RzCore *r, RzBinFile *binfile) {
 	rz_config_set(r->config, "file.type", info->rclass);
 	rz_config_set(r->config, "cfg.bigendian",
 		info->big_endian ? "true" : "false");
-	if (!info->rclass || strcmp(info->rclass, "fs")) {
+	if (!info->rclass) {
```

Comment:
```suggestion
	if (!info->rclass || (info->rclass && strcmp(info->rclass, "fs"))) {
```
I guess this line

Reply:
```suggestion
	if (!info->rclass || strcmp(info->rclass, "fs")) {
```
I think the original line was good as it was. Why did you remove the strcmp?